projects/1.20/assets/game-discs/gamediscs/lang/zh_cn.json

```
@@ -0,0 +1,51 @@
+{
+    "creativetab.game_discs_tab": "游戏光碟",
+  
+  
+    "gamediscs.flappy_bird": "飞翔的蜜蜂",
+    "gamediscs.slime": "贪吃史莱姆",
+    "gamediscs.blocktris": "消除方块",
+    "gamediscs.tnt_sweeper": "TNT扫雷",
+    "gamediscs.pong": "乒乓",
+    "gamediscs.froggie": "蛙跃水沟",
+    "gamediscs.pong_game": "乒乓",
+  
+  
+    "item.gamediscs.gaming_console": "游戏机",
+    "item.gamediscs.redstone_circuit": "红石电路",
+    "item.gamediscs.processor": "处理器",
+    "item.gamediscs.battery": "电池",
+    "item.gamediscs.display": "屏幕",
+    "item.gamediscs.control_pad": "控制器",
+    "item.gamediscs.game_disc_flappy_bird": "游戏光碟",
+    "item.gamediscs.game_disc_slime": "游戏光碟",
+    "item.gamediscs.game_disc_blocktris": "游戏光碟",
+    "item.gamediscs.game_disc_tnt_sweeper": "游戏光碟",
+    "item.gamediscs.game_disc_pong": "游戏光碟",
+    "item.gamediscs.game_disc_froggie": "游戏光碟",
+  
+  
+    "gui.gamingconsole.title": "游戏机",
+    "gui.gamingconsole.press_any_key": "请按下任意键",
+    "gui.gamingconsole.select_game": "选择游戏",
+    "gui.gamingconsole.score": "得分",
+    "gui.gamingconsole.best_score": "最佳",
+    "gui.gamingconsole.new_best_score": "新纪录",
+    "gui.gamingconsole.won": "你赢了！",
+    "gui.gamingconsole.died": "你输了！",
+    "gui.gamingconsole.hold": "暂存",
+    "gui.gamingconsole.next": "下一个",
+  
+  
+    "sounds.gamediscs.jump": "游戏机：跳跃",
+    "sounds.gamediscs.click": "游戏机：按下按钮",
+    "sounds.gamediscs.point": "游戏机：得分",
+    "sounds.gamediscs.new_best": "游戏机：新纪录",
+    "sounds.gamediscs.game_over": "游戏机：游戏结束",
+    "sounds.gamediscs.select": "游戏机：选择",
+    "sounds.gamediscs.confirm": "游戏机：确认",
+    "sounds.gamediscs.explosion": "游戏机：消除",

```

```suggestion
    "sounds.gamediscs.explosion": "游戏机：瞬降",
```
经检验，降落后依然可以移动方块，也即不锁定，因此建议改为这个
https://tetris.huijiwiki.com/wiki/%E9%99%8D%E8%90%BD#%E7%A1%AC%E9%99%8D